WalletCore/Classes/ViewModel/BaseAssetsViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  BaseAssetsViewModel.swift
+//  WalletCore
+//
+//  Created by Lyubomir Marinov on 23.01.18.
+//  Copyright © 2018 Lykke. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+
+open class BaseAssetsViewModel {
+    public var assetsViewModel: AssetsViewModel!
+    
+    /// Standart loading view model
+    public let loadingViewModel: LoadingViewModel
+    
+    /// Standart error driver
+    public let errors: Driver<[AnyHashable: Any]>
+    
+    public init(authManager: LWRxAuthManager = LWRxAuthManager.instance)
+    {
+        let result = authManager.allAssets.request().filterSuccess()
+            .throttle(1, scheduler: MainScheduler.instance)
```

Comment:
What's de benefit of using throttle in this case ? Usually this operator is used when there is a user interaction.